src/sass/form-updates.scss

```diff
@@ -0,0 +1,71 @@
+.has-feedback {
```

Comment:
Unfortunately this class names are not very semantic, *feedback* has little to do with search. Could you refactor to use semantic names and rules without nesting?